frontend/src/components/generic/CategoryChip.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <v-chip dark :color="cat.color" v-bind="$attrs" v-on="$listeners">
```

Comment:
```suggestion
  <v-chip dark :color="cat.color" :text-color="textColor" v-bind="$attrs" v-on="$listeners">
```

Ich würde das gerade ohne zusätzliches div lösen

Reply:
thx